podcast-player/podcast-player-app/src/components/episode/EpisodeModal.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+import { ReactComponent as PlayButton } from "../../assets/SVG/play2.svg";
+import { ReactComponent as CloseButton } from "../../assets/SVG/cross.svg";
+
+const Backdrop = (props) => {
+  return <div className="backdrop" onClick={props.onConfirm} />;
+};
+
+const ModalOverlay = ({
```

Comment:
Interesting that you implemented your own modal. Finding a good component library can be hard.

Modals are notoriously difficult to get right. A good library will handle complex cases out of the box such as keyboard focus trapping and smaller screen sizes (eg: mobile)